docs/src/main/sphinx/admin.md

```diff
@@ -21,6 +21,17 @@ admin/fault-tolerant-execution
 (admin-event-listeners)=
 ## Event listeners
 
+Event listeners are plugins that allow streaming of query events to an external
+system. By default, query events are only kept in memory for a limited number of
```

Comment:
This is inaccurate. Query events are not kept in memory or controlled by `query.max-history`.

That option is to control how many queries are tracked for display in the UI.

The problem with how this is written is that it mixes "query events" (query creation, query completion, split processed) with the query state that's displayed in the UI.

Reply:
Ok.. I have to unfurl this somehow then .. let me work on that after the internal communication stuff